support-frontend/assets/components/paymentMethodSelector/paymentMethodSelector.tsx

```diff
@@ -97,6 +97,7 @@ export function PaymentMethodSelector({
 		<div css={container}>
 			<PaymentMethodSelectorLegend />
 			<RadioGroup
+				id="paymentMethod"
```

Comment:
Adding an id is great here, could also add `role='radiogroup'` which hadn't been ported over initially

Reply:
Will do that in my next PR!